pkg/registry/populator_test.go

```diff
@@ -1272,19 +1271,14 @@ func TestOverwrite(t *testing.T) {
 					true).Populate(registry.ReplacesMode)
 			}
 			require.NoError(t, populate(tt.args.firstAdd, nil))
-			popErr := populate(tt.args.secondAdd, tt.args.overwrites)
-			if agg, ok := popErr.(utilerrors.Aggregate); ok {
-				// The order of the errors that
-				// comprise an aggregate error isn't
-				// important to the tested behaviors,
-				// so sort them:
-				errs := agg.Errors()
-				sort.Slice(errs, func(i, j int) bool {
-					return errs[i].Error() < errs[j].Error()
-				})
-				popErr = utilerrors.NewAggregate(errs)
+
+			err = populate(tt.args.secondAdd, tt.args.overwrites)
```

Comment:
I ran into a similar issue, but needed to make it even less dependent on a specific error type, since I ended up flattening the array that was being compared under the hood.